opentasks/src/main/java/org/dmfs/tasks/model/TimeZoneChoicesAdapter.java

```diff
@@ -196,10 +196,9 @@ private String getGMTOffsetString(long millis)
         long absmillis = (millis < 0) ? -millis : millis;
         int minutes = (int) ((absmillis / (1000 * 60)) % 60);
         int hours = (int) ((absmillis / (1000 * 60 * 60)) % 24);
-        StringBuilder builder = new StringBuilder("(GMT");
-        builder.append((millis >= 0) ? '+' : '-');
-        builder.append(String.format("%02d:%02d", hours, minutes)).append(") ");
-        return builder.toString();
+        String builder = "(GMT" + ((millis >= 0) ? '+' : '-') +
+                String.format("%02d:%02d", hours, minutes) + ") ";
+        return builder;
```

Comment:
Looking at this now, I think we're probably better off with something like this:
```
return String.format("(GMT%c%02d:%02d) ", millis >= 0 ? '+' : '-', hours, minutes)
```
i.e. just `String.format` the entire thing.